assets/js/src/application.js

```diff
@@ -115,7 +115,9 @@
         },
         transformData: function (hits) {
           return hits.map(function (hit) {
+            var baseurl = document.getElementById('search-input').dataset.baseurl
```

Comment:
Since IE 10 is still supported, please use `getAttribute` because it doesn't support `dataset`.

Also, did you verify everything works locally? I haven't had the time to test it yet.

Reply:
Yes, it still works.